packages/server/src/arpa_reporter/routes/uploads.js

```diff
@@ -155,6 +155,32 @@ router.post('/:id/validate', requireUser, async (req, res) => {
     }
 });
 
+router.post('/:id/invalidate', requireUser, async (req, res) => {
```

Comment:
we should update this to `requireAdminUser` but since this is not visible to any users at the moment and we have audit-log enabled, lets update this in a follow-up PR